packages/tags/src/Tag.ts

```diff
@@ -129,12 +129,9 @@ export class Tag extends SpectrumElement {
         if (!this.hasAttribute('role')) {
             this.setAttribute('role', 'listitem');
         }
-        this.setAttribute(
-            'tabindex',
-            !this.disabled && this.matches(':first-of-type:not([disabled])')
-                ? '0'
-                : '-1'
-        );
+        if (!this.disabled) {
+            this.setAttribute('tabindex', this.deletable ? '0' : '-1');
```

Comment:
Great start! 

This previously applied a pattern called "floating tab index" wherein only one time in the `sp-tags` element at a time should have `tabindex=0`, which was powered by `:first-of-type:not([disabled])` in the previous code. That might not be the best approach, but we _should_ continue to surface that pattern. The fact that removing it didn't cause a test to fail also means we likely want to add a test to support the continued presence of the "floating tab index" pattern.

Please update the `tabindex` management here and add a test to ensure that such a pattern is not regressed.